packages/@webex/plugin-meetings/src/reachability/index.ts

```diff
@@ -152,19 +154,23 @@ export default class Reachability extends EventsScope {
 
     const matchingReachedClusters = Object.values(this.clusterReachability).reduce(
       (acc, cluster) => {
-        const reachedSubnetsArray = Array.from(cluster.reachedSubnets);
+        const detailsArrays = [
+          ...(cluster.getResult().udp.details || []),
+          ...(cluster.getResult().tcp.details || []),
+          ...(cluster.getResult().xtls.details || []),
+        ];
 
         let logMessage = `Reachability:index#isSubnetReachable --> Cluster ${cluster.name} reached [`;
-        for (let i = 0; i < reachedSubnetsArray.length; i += 1) {
-          const subnet = reachedSubnetsArray[i];
-          const reachedSubnetFirstOctet = subnet.split('.')[0];
+        for (let i = 0; i < detailsArrays.length; i += 1) {
+          const subnet = detailsArrays[i];
+          const reachedSubnetFirstOctet = subnet.serverIp.split('.')[0];
 
           if (selectedSubnetFirstOctet === reachedSubnetFirstOctet) {
             acc.add(cluster.name);
           }
 
-          logMessage += `${subnet}`;
-          if (i < reachedSubnetsArray.length - 1) {
+          logMessage += `${JSON.stringify(subnet)}`;
```

Comment:
we don't need to stringify the whole struct here, we just need the `serverIp` field